inference-engine/tests/functional/plugin/conformance/test_runner/api_conformance_runner/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ addIeTargetTest(
         ADDITIONAL_SOURCE_DIRS
             ${CMAKE_CURRENT_SOURCE_DIR}/src
```

Comment:
do we need the renaming above `avior/executable_network/get_metric.cpp → ...r/executable_network/core_integration.cpp `?

Reply:
will be fixed, thanks